test-app/tsconfig.json

```diff
@@ -1,21 +1,70 @@
 {
   "compilerOptions": {
-    "target": "es2020",
-    "allowJs": true,
+    "target": "ES2021",
```

Comment:
these changes came from `ember install ember-cli-typescript` as were discussed in #296